### PR TITLE
Makefile improvements

### LINKED ISSUE #302: Include cmdlet build in the overall makefile

### PR DESCRIPTION
**Pull request by Jim3Things**
- Filter out unit test support files from the production files list
- Add powershell cmdlet build into the general makefile.
- Add script_test command to cover tests that need a running cloudchamber service to execute.

closes #302

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -205,13 +210,15 @@ SRC_ARTIFACTS = \
...
 
-
 ARTIFACTS = $(addprefix $(KIT_BUILD)/, $(notdir $(SRC_ARTIFACTS)))
 
 OBSERVER = \
 	$(OBSERVER_UI_BUILD)/asset-manifest.json \
 	$(OBSERVER_UI_BUILD)/index.html
 
+CMDLETS_OUT = \
+	$(CMDLETS_BUILD)/Debug/netstandard2.0/CloudChamber.Cmdlets.dll
```
Comment (MikeCJohnsonUk) [inline Makefile:220]: Should this really be "Debug" in the path?

File: clients/powershell/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+CP = cp
+CP-RECURSIVE = $(CP) -r
+
+MD = mkdir -p
+RM-RECURSIVE = $(RM) -r
+
+TOUCH = touch --no-create
+
```
Comment (MikeCJohnsonUk) [inline clients/powershell/Makefile:8]: I'm beginning to think we are accumulating enough boilerplate that it'll be worth having a standard include file. What do you think? 

Not as part of this change, but I'd be interested in opinions.

File: Makefile (hunk 1 above)
Comment (Jim3Things) [reply Makefile:220]: It's what we're building at the moment.  When we've figured out how to do the packaging correctly, then I'm assuming that we'll cut over to a Release build.  I've just been holding off on going to a release build before then.

File: clients/powershell/Makefile (hunk 2 above)
Comment (Jim3Things) [reply clients/powershell/Makefile:8]: Yeah, I agree.  I was thinking the same thing.

File: Makefile (hunk 1 above)
Comment (MikeCJohnsonUk) [reply Makefile:220]: Ok.

File: clients/powershell/Makefile (hunk 2 above)
Comment (MikeCJohnsonUk) [reply clients/powershell/Makefile:8]: I've added issue #315

**Review by MikeCJohnsonUk — APPROVED** [review]

**Pull request merged**